src/rgw/rgw_admin.cc

```diff
@@ -193,9 +193,12 @@ void _usage()
   cout << "   --master-zone=<id>        master zone id\n";
   cout << "   --rgw-realm=<realm>       realm name\n";
   cout << "   --realm-id=<realm id>     realm id\n";
-  cout << "   --realm-new-name=<realm new name> realm new name\n";
-  cout << "   --rgw-zonegroup=<zonegroup>   zonegroup name\n";
-  cout << "   --zonegroup-id=<zonegroup id> zonegroup id\n";
+  cout << "   --realm-new-name=<realm new name>\n";
+  cout << "                             realm new name\n";
+  cout << "   --rgw-zonegroup=<zonegroup>\n";
+  cout << "                             zonegroup name\n";
+  cout << "   --zonegroup-id=<zonegroup id>\n";
+  cout << "                             zonegroup id\n";
```

Comment:
It's kind of sad that we waste so much space with `<realm new name>`. it doesn't actually give any new information, just repeats the flag's name. it might be worthwhile to go through and just use `<id>` or `<name>` for all of these flags instead. what do you think?